libtiff/libtiff_ctypes.py

```diff
@@ -1210,6 +1210,20 @@ def NumberOfStrips(self):
         return libtiff.TIFFNumberOfStrips(self).value
     numberofstrips = NumberOfStrips
 
+    @debug
+    def WriteScanline(self, buf, row):
```

Comment:
Add the `sample=0` argument to the method and update the function call accordingly.

Reply:
Good catch! I've updated accordingy; we've just never had a use case for a non-0 sample argument.